mandb.if

```diff
@@ -122,14 +122,19 @@ interface(`mandb_manage_cache_content',`
 #
 interface(`mandb_admin',`
 	gen_require(`
-		type mandb_t, mandb_cache_t;
+		type mandb_t, mandb_cache_t, mandb_unit_t;
 	')
 
 	allow $1 mandb_t:process { ptrace signal_perms };
 	ps_process_pattern($1, mandb_t)
 
 	mandb_run($1, $2)
 
-	# pending
-	# miscfiles_manage_man_cache_content(mandb_t)
+	ifdef(`init_systemd',`
+		gen_require(`
+			class service { start status stop };
+		')
+
+		allow $1 mandb_unit_t:service { start status stop };
```

Comment:
probably, but `init_start_stop_service()` requires an initrc_script type
btw, there is no systemd unit file matching `/usr/lib/systemd/system/[^/]*man-db.*` on debian sid, so I added this statement just for fashion